src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/DynamicContentTypeBuilder.cs

```diff
@@ -12,27 +12,38 @@ namespace OrchardCore.ContentManagement.GraphQL.Queries.Types
 {
     public class DynamicContentTypeBuilder : IContentTypeBuilder
     {
+        // If registered, then fields input should be exposed, otherwise ignored.
+        public class FieldsInputMarker
+        {
+        }
+
         private readonly IHttpContextAccessor _httpContextAccessor;
         private readonly GraphQLContentOptions _contentOptions;
         protected readonly IStringLocalizer S;
+        private readonly FieldsInputMarker _fieldsInputMarker;
+
         private readonly Dictionary<string, FieldType> _dynamicPartFields;
 
         public DynamicContentTypeBuilder(IHttpContextAccessor httpContextAccessor,
             IOptions<GraphQLContentOptions> contentOptionsAccessor,
-            IStringLocalizer<DynamicContentTypeBuilder> localizer)
+            IStringLocalizer<DynamicContentTypeBuilder> localizer,
+            FieldsInputMarker fieldsInputMarker = null)
```

Comment:
Do not declare this as null, refer to lines 42-43 of the current class file.

Reply:
Done